src/theory/quantifiers/sygus/ce_guided_conjecture.cpp

```diff
@@ -163,10 +164,19 @@ void CegConjecture::assign( Node q ) {
   }
   
   // initialize the guard
-  if (d_ceg_si->getGuard().isNull())
+  d_feasible_guard = nm->mkSkolem("G", nm->booleanType());
+  d_feasible_guard = Rewriter::rewrite(d_feasible_guard);
```

Comment:
what is the reason again to rewrite fresh boolean constants?

Reply:
OutputChannel's ensureLiteral has the invariant that its input must be in rewritten form, otherwise we crash. While rewriting a Boolean skolem here will almost assuredly do nothing, this is a sanity check.

Reply:
ok